app/templates/welcome.js

```diff
@@ -1,5 +1,7 @@
 const html = require('choo/html');
 const assets = require('../../common/assets');
+const bytes = require('../utils');
+const config = require('../../server/config');
 const fileList = require('./fileList');
 const { fadeOut } = require('../utils');
```

Comment:
add `bytes` here `const { bytes, fadeOut } = require('../utils');` instead of a separate `require`